py/desispec/desi_proc_dashboard.py

```diff
@@ -161,7 +161,8 @@ def main(args):
 
     if args.output_dir is None:
         if 'DESI_DASHBOARD' not in os.environ.keys():
-            os.environ['DESI_DASHBOARD']=os.environ["HOME"]
+            os.environ['DESI_DASHBOARD']=os.path.join(
```

Comment:
This makes sense in that the normal calls used to monitor any given prod use this path structure and it allows us to run the script without having to specify the path.

The old setting was chosen so that anyone could create a custom webpage for any prod with a default that wouldn't crash due to lack of write access or overwrite the existing dashboard page (if run as desi user with the proper permissions). E.g. making a page for specific nights or object types while letting a cron job continue updating the full dashboard of the prod.

This change simplifies the standard use case by the pipeline and makes our lives easier the majority of the time, while making it slightly harder/more dangerous when run by a non-power user.

Thoughts?

(To be clear: I don't feel strongly in either direction. I'm just giving the context.)

Reply:
Discussed with Anthony.  Having a non-desi user make a dashboard and forget to specify an alternate output is "harmless" because it can't overwrite anything.  We'll need to be a bit more careful when running custom dashboards as the desi user, but the worse case scenario is we overwrite the dashboard and have to run it again with default parameters, which is fairly fast.  I.e. we'll leave this update in place.